apps/odo/mix.exs

```diff
@@ -38,8 +38,8 @@ defmodule Odo.MixProject do
       {:libvault, "~> 0.2"},
       {:poison, "~> 3.1", override: true},
       {:redix, "~> 0.10"},
-      {:retry, "~> 0.13.0"},
-      {:smart_city, "~> 3.0"},
```

Comment:
is there a reason to set this to 3.5?  If you leave it as 3.0 it should do the right thing as long as the dependency is unlocked.

Reply:
I'm not sure, I didn't make the change. Might have come in via a merge?